src/main/java/io/github/opencubicchunks/cubicchunks/core/CubicChunksConfig.java

```diff
@@ -180,7 +187,14 @@ public static void flipForceCubicChunks() {
                 forceLoadCubicChunks = ForceCCMode.NEW_WORLD;
                 break;
             case NEW_WORLD:
-                forceLoadCubicChunks = ForceCCMode.NONE;
+                ResourceLocation nextType = CubeGeneratorsRegistry.getNextGeneratorType(new ResourceLocation(compatibilityGeneratorType));
+                if (nextType == null) {
+                    forceLoadCubicChunks = ForceCCMode.NONE;
+                    compatibilityGeneratorType = CubeGeneratorsRegistry.getFirstGeneratorType().toString();
```

Comment:
Since this hack won't be possible with forge registries:
After the registry is frozen (it definitely is at postInit), just create a list with all the possible options in the GUI.